holoviews/plotting/bokeh/element.py

```diff
@@ -1256,16 +1252,13 @@ def _init_glyph(self, plot, mapping, properties):
             del properties['legend_label']
 
         # ALERT: This only handles XYGlyph types right now
+        mapping = property_to_dict(mapping)
         if 'x' in mapping:
```

Comment:
Just realized that this was introduced as a fix after unit tests were already passing so the fact that unit tests are passing locally with this change means nothing. I suppose if the unit tests for Bokeh 2 start passing that will at least be somewhat informative...

Reply:
Bokeh 2 unit tests are now passing. The question now is whether the test that was failing in examples remains fixed with this change. At least we can see that the issue with Bokeh 2 is related to this code.